templates/web/base/admin/templates_index.html

```diff
@@ -0,0 +1,11 @@
+[% INCLUDE 'admin/header.html' title=loc('Response Templates') -%]
+
+<ul>
+    [% FOR body IN bodies %]
+        <li>
+            <a href="/admin/templates/[% body.id %]/[% t.id %]">[% body.name %]</a>
```

Comment:
You don't have a template ID at this point. Do we want to duplicate the body page here, or should it add a Templates link to the existing body page instead, and this page could be, dunno, latest created templates, or blank, or something exciting.